### PR TITLE
feat: breadcrumbs

### PR DESCRIPTION
**Pull request by synox**
Test URLs:
- https://breadcrumb--helix-block-collection--adobe.hlx.page/block-collection/sample-pages/speed
- https://breadcrumb--helix-block-collection--adobe.hlx.page/block-collection/headings
- https://breadcrumb--helix-block-collection--adobe.hlx.page/block-collection/hero (not in nav)

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -273,3 +273,50 @@ header nav .nav-sections ul > li > ul > li {
...
+/* stylelint-disable-next-line no-descending-specificity */
+header .breadcrumbs ol li {
+  padding-right: 8px;
+  padding-inline-start: 0;
+  opacity: 0.7;
+}
+
+header .breadcrumbs ol li:not(:last-child)::after {
+  content: "/";
+  padding-left: 8px;
```
Comment (fkakatie) [inline blocks/header/header.css:307]:
```suggestion
  padding-left: 1ch;
```

File: blocks/header/header.css [hunk 2]
```diff
@@ -273,3 +273,50 @@ header nav .nav-sections ul > li > ul > li {
...
+}
+
+header .breadcrumbs ol {
+  display: flex;
+  flex-flow: wrap;
+  list-style: none;
+  padding: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```
Comment (fkakatie) [inline blocks/header/header.css:296]:
```suggestion
  gap: 1ch;
}
```

File: blocks/header/header.css [hunk 3]
```diff
@@ -273,3 +273,50 @@ header nav .nav-sections ul > li > ul > li {
...
+
+/* stylelint-disable-next-line no-descending-specificity */
+header .breadcrumbs ol li {
+  padding-right: 8px;
```
Comment (fkakatie) [inline blocks/header/header.css:300]:
```suggestion
```

File: blocks/header/header.js [hunk 4]
```diff
@@ -86,11 +88,70 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
...
+  const homeUrl = document.querySelector('.nav-brand a').href;
+  crumbs.unshift({ title: homePlaceholder, url: homeUrl });
+
+  // last link is current page and should not be linked
+  crumbs[crumbs.length - 1].url = null;
+  return crumbs;
+}
+
+async function buildBreadcrumbs() {
+  const breadcrumbs = document.createElement('div');
```
Comment (fkakatie) [inline blocks/header/header.js:126]:
```suggestion
  const breadcrumbs = document.createElement('nav');
```

File: styles/styles.css [hunk 5]
```diff
@@ -37,8 +37,11 @@
   --heading-font-size-s: 20px;
   --heading-font-size-xs: 18px;
 
-  /* nav height */
+  /* header */
```
Comment (synox) [inline styles/styles.css:40]: There is still a layout shift

**Review by fkakatie — APPROVED** [review]

**Pull request merged**